engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheHashingJobActorDataSpec.scala

```diff
@@ -0,0 +1,94 @@
+package cromwell.engine.workflow.lifecycle.execution.callcaching
+
+import cromwell.backend._
+import cromwell.backend.standard.callcaching.StandardFileHashingActor.SingleFileHashRequest
+import cromwell.core.TestKitSuite
+import cromwell.core.callcaching._
+import cromwell.engine.workflow.lifecycle.execution.callcaching.CallCacheHashingJobActor.{CallCacheHashingJobActorData, CompleteFileHashingResult, NoFileHashesResult, PartialFileHashingResult}
+import org.scalatest.concurrent.Eventually
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.{FlatSpecLike, Matchers}
+
+class CallCacheHashingJobActorDataSpec extends TestKitSuite with FlatSpecLike with BackendSpec with Matchers with Eventually with TableDrivenPropertyChecks {
+  behavior of "CallCacheReadingJobActorData"
+
+  val fileHash1 = HashResult(HashKey("key"), HashValue("value"))
+  val fileHash2 = HashResult(HashKey("key2"), HashValue("value2"))
+  val fileHash3 = HashResult(HashKey("key3"), HashValue("value3"))
+  val fileHashRequest1 = mock[SingleFileHashRequest]
```

Comment:
`SingleFileHashRequest` should be a `final case class` and so unmockable. 

That's ok, because you don't need to mock a case class, since this is a test we can just use `SingleFileHashRequest(null, hashKey, null null)` and allow the NPE if we set up the test wrong.